src/common/color-element.js

```diff
@@ -80,6 +89,23 @@ const Self = class ColorElement extends NudeElement {
 		}
 
 		customElements.define(this.tagName, this);
+
+		if (this.globalStyle) {
+			// NudeElement imports global styles using `@import url()` inside an `<style>` element with a `data-for` attribute equal to the element's tag name
```

Comment:
Yeah, looks like this was a bit overfit. We should definitely support verbatim styles being included, not just URLs. We could probably even detect the two so the user can just give us an array of strings

Reply:
That's a very good idea. With something like this do: https://github.com/nudeui/element/pull/43?